WordPress/Classes/ViewRelated/Post/PostSettingsSelectionViewController.h

```diff
@@ -1,10 +1,18 @@
 #import <UIKit/UIKit.h>
 
+extern NSString * const SettingsSelectionTitleKey;
+extern NSString * const SettingsSelectionTitlesKey;
+extern NSString * const SettingsSelectionValuesKey;
+extern NSString * const SettingsSelectionDefaultValueKey;
+extern NSString * const SettingsSelectionCurrentValueKey;
+
 @interface PostSettingsSelectionViewController : UITableViewController
 
-@property (nonatomic, copy) void(^onItemSelected)(NSString *);
+@property (nonatomic, copy) void(^onItemSelected)(NSObject *);
```

Comment:
Ah, yes. Its less clear with the PR divided as it is. The value type doesn't particular matter and should be generic. NSString was too restrictive.  This change allows for an NSDictionary or NSNumber